go.mod

```diff
@@ -7,7 +7,9 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.34.0
 	github.com/kr/pretty v0.3.1
-	github.com/vmware/go-vcloud-director/v2 v2.26.0
+	github.com/vmware/go-vcloud-director/v3 v3.0.0-alpha.2
+	// TODO: TM: Change this to v4
+	github.com/vmware/terraform-provider-vcd/v3 v3.14.0
```

Comment:
Updating `main.go` to use `v4` should get rid of this import alltogether

Reply:
True, fixed